velox/exec/Operator.h

```diff
@@ -361,7 +382,18 @@ class Operator {
     return false;
   }
 
-  OperatorStats& stats() {
+  /// Returns copy of operator stats, optionally clearing them inside the
+  /// operator after retrieval.
+  OperatorStats stats(bool clear);
```

Comment:
```
OperatorStats stats(bool clear = false);
```

Reply:
Default values for arguments = hidden bugs. :)